src/JetNtuplizer.cc

```diff
@@ -131,6 +131,11 @@ JetNtuplizer::Analyze( const edm::Event& iEvent, const edm::EventSetup& iSetup )
     JetInfo.PtCorrRaw   [JetInfo.Size] = it_jet->correctedJet( "Uncorrected" ).pt();
     JetInfo.PtCorrL2    [JetInfo.Size] = it_jet->correctedJet( "L2Relative" ).pt();// L2(rel)
     JetInfo.PtCorrL3    [JetInfo.Size] = it_jet->correctedJet( "L3Absolute" ).pt();// L3(abs)
+    JetInfo.NCH         [JetInfo.Size] = it_jet->chargedMultiplicity();
+    JetInfo.CEF         [JetInfo.Size] = it_jet->chargedEmEnergyFraction();
+    JetInfo.NHF         [JetInfo.Size] = it_jet->neutralHadronEnergyFraction();
+    JetInfo.NEF         [JetInfo.Size] = it_jet->neutralEmEnergyFraction();
+    JetInfo.CHF         [JetInfo.Size] = it_jet->chargedHadronEnergyFraction();
```

Comment:
Please also remove the corresponding lines in 
https://github.com/sam7k9621/bprimeKit/blob/dbcbbe340359694477b4072c5f0f68b498eaa12e/src/JetNtuplizer.cc#L233